src/components/FeatureViews/Dashboard/Dashboard.js

```diff
@@ -1,54 +1,162 @@
-import {connect} from "react-redux";
-import React, {Component} from "react";
-import "./Dashboard.scss";
-
-import Grid from "@material-ui/core/Grid";
-import Card from "@material-ui/core/Card";
-import CardActionArea from "@material-ui/core/CardActionArea";
-import CardActions from "@material-ui/core/CardActions";
-import CardContent from "@material-ui/core/CardContent";
-import CardMedia from "@material-ui/core/CardMedia";
-import Button from "@material-ui/core/Button";
-import Typography from "@material-ui/core/Typography";
-
-class Dashboard extends Component {
-	render() {
-		return (
-			<div className="`Dashboard`">
-				<h1>Dashboard</h1>
-				<Grid container spacing={2} className="Root">
-					<Card>
-						<CardActionArea>
-							<CardMedia></CardMedia>
-							<CardContent>
-								<Typography gutterBottom variant="h5" component="h2">
-									Contact Us
-								</Typography>
-							</CardContent>
-						</CardActionArea>
-						<CardActions>
-							<Button size="small" color="primary">
-								Share
-							</Button>
-							<Button size="small" color="primary">
-								Learn More
-							</Button>
-						</CardActions>
-					</Card>
-				</Grid>
-			</div>
-		);
-	}
-}
-
-Dashboard.propTypes = {};
-
-function mapStateToProps(state) {
-	return {};
-}
-
-function mapDispatchToProps(dispatch) {
-	return {};
-}
-
-export default connect(mapStateToProps, mapDispatchToProps)(Dashboard);
+import { useSelector } from 'react-redux';
+import { useQuery } from '@apollo/react-hooks';
+import gql from 'graphql-tag';
+import React, {useState} from 'react';
+import {Link} from 'react-router-dom';
+import './Dashboard.scss';
+import Today from './Today';
+import UnpaidSessions from './../AdminPortal/UnpaidSessions';
+import Loading from "components/Loading";
+
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Typography from '@material-ui/core/Typography';
+import Paper from "@material-ui/core/Paper";
+import DashboardNotes from './DashboardNotes';
+import moment from 'moment';
+import Moment from 'react-moment';
+import Select from 'react-select';
+import { makeStyles } from "@material-ui/styles";
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        [theme.breakpoints.down('md')]: {
+            padding: "0px"
+        }
+    },
+    date: {
+        [theme.breakpoints.down('md')]: {
+            fontSize: "20px"
+        }
+    },
+    todayCard: {
+        [theme.breakpoints.down('md')]: {
+            fontSize: "6px"
+        }
+    }
+}))
+
+const Dashboard = () => {
+    const classes = useStyles();
+    const user = useSelector(({auth}) => auth) || [];
```

Comment:
`auth` doesn't store all the user details, nor is it an array - it is an object formatted like so:

    {
        token
        email
        accountType
    }

You'll have to fetch other details yourself, either via the `admin` or `userInfo` fields.